### DISCUSSION TITLE
Tests are duplicated instead of being placed on history and retries

### DISCUSSION THREAD

**Discussion by surajsahu8** [allure-framework/allure2#1866] [Bug Reports]
In [v2.0.0-beta.21](https://www.npmjs.com/package/allure-playwright/v/2.0.0-beta.21) - Tests are duplicated instead of being placed on history and retries.

**Comment by cheshi-mantu** [comment]
Hi @surajsahu8 
if you considering the behaviour you are facing is a bug, please file a bug report **with the description and evidence** [here](https://github.com/allure-framework/allure-js/issues)

thanks
